imgui_impl_vitagl.cpp

```diff
@@ -25,7 +29,7 @@ uint8_t *gColorBuffer = nullptr;
 uint16_t *gIndexBuffer = nullptr;
 uint32_t gCounter = 0;
 
-bool touch_usage = false;
+bool touch_usage = true;
```

Comment:
Ok

Reply:
Another question: do you want the default front touch behavior to be “jump to finger” (direct mode) or “drag pointer with finger” (indirect mode like on a laptop touchpad). 

I prefer indirect mode because it is easier to hit the small buttons etc. and the finger is not in the way.